MPF.Core/UI/ViewModels/MainViewModel.cs

```diff
@@ -76,6 +76,20 @@ public Data.Options Options
 
         #region Properties
 
+        /// <summary>
+        /// Indicates the inability to create IRDs
+        /// </summary>
+        public bool CannotCreateIRD
```

Comment:
Is there a reason that you're using a negative state here?

Reply:
It binds to whether the "Create PS3 IRD" button in the tool menu is enabled or not.
New property is easier than trying to do negatives in XAML

Reply:
Ah, sorry, it is bound to the tooltip being enabled!
The button is bound to CreateIRDMenuItemEnabled
If there was an easy way to negate CreateIRDMenuItemEnabled in XAML then I would do that

Reply:
If you just gate the functionality behind net6.0+, you won't need a special tooltip. The button just wouldn't be there.